adxl345/example/examples.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2023 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package example
+
+import (
+	"fmt"
+	"log"
+	"periph.io/x/conn/v3/i2c/i2creg"
+	"periph.io/x/conn/v3/spi/spireg"
+	"periph.io/x/devices/v3/adxl345"
+	"periph.io/x/host/v3"
+	"time"
+)
+
+// I2C uses an adxl345 device connected by I²C.
+// it reads the acceleration values every 30ms for 30 seconds.
+// You can i use `i2dctools`  to find the I²C bus number
+// e.g : sudo apt-get install i2c-tools
+//
+//	sudo i2cdetect -y 1
+func I2C(addr uint16) {
```

Comment:
You want it to be ExampleNewI2C() for it to show up at the right location in the documentation. See https://go.dev/blog/examples for more information.

Reply:
Thank you for pointing out "testable examples"  in fact My IDE was pointing out that the signature was not acceptable in the test package (see below) . In fact that why i created an "example" package.  I've been following your recommandation in my last commit. 
<img width="588" alt="Screenshot 2024-01-02 à 16 24 33" src="https://github.com/periph/devices/assets/963307/fd66a8ac-38b9-496a-92ac-95a2c68e6fb0">